spec/ParseWebSocketServer.spec.js

```diff
@@ -75,6 +76,32 @@ describe('ParseWebSocketServer', function () {
     expect(wssError).toBe('Invalid Packet');
   });
 
+  it('closes interrupted connection', async () => {
+    const onConnectCallback = jasmine.createSpy('onConnectCallback');
+    const http = require('http');
+    const server = http.createServer();
+    const parseWebSocketServer = new ParseWebSocketServer(server, onConnectCallback, {
+      websocketTimeout: 5,
+    }).server;
+    const ws = new EventEmitter();
+    ws.readyState = 0;
+    ws.OPEN = 0;
+    ws.ping = jasmine.createSpy('ping');
+    ws.terminate = jasmine.createSpy('terminate');
+
+    parseWebSocketServer.onConnection(ws);
+
+    // Make sure callback is called
+    expect(onConnectCallback).toHaveBeenCalled();
+    await new Promise(resolve => setTimeout(resolve, 10));
+    // Make sure we ping to the client
+    expect(ws.ping).toHaveBeenCalled();
+    await new Promise(resolve => setTimeout(resolve, 10));
+    // make sure connection close after we didn't answer the ping on time
```

Comment:
Could you remove these comments? I think the code is self-explanatory

Reply:
Done :)